### ISSUE TITLE
Scrolling works via the mouse but not via the keyboard

### ISSUE THREAD

**Issue by alex-bronze** [cerner/terra-core#2322]
# Bug Report

## Description
On many Terra-generated UIs (including the Terra documentation itself), it is only possible to scroll by using the mouse but _not_ the keyboard. The standard navigational keys are ignored.

Link: https://engineering.cerner.com/terra-ui/#/components/terra-action-header/action-header/action-header

## Steps to Reproduce

1. Go to any Terra-generated UI that has a scroll bar, including the Terra documentation itself (https://engineering.cerner.com/terra-ui/#/components/terra-action-header/action-header/action-header)
2. Try scrolling via the mouse. This works.
3. Try using the standard navigational keys. This does not work.
    - Up/Down
    - PageUp/PageDown
    - Home/End

## Expected Behavior
The standard navigational keys should work according to their definitions.

## Environment

* Component Name and Version:
* Browser Name and Version: **Chrome, Firefox, Edge**
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile): **Windows 10, macOS**

**Comment by bjankord** [comment]
I'm able to reproduce this in Chrome, Firefox, and Safari on Mac. I've narrowed down the issue to the use of `tabindex="-1"` in 2 places. One on the `main` element and one on the overlay div 2 elements above the main element.

<img width="1072" alt="Screen Shot 2019-04-03 at 8 23 46 PM" src="https://user-images.githubusercontent.com/633148/55523372-70b72780-564e-11e9-8f0a-412f1aae1e65.png">

Removing `tabindex="-1"` from those 2 DOM elements seems to restore keyboard page scrolling. Need to figure out what those tab index attributes were used for before we remove them though.

**Comment by JakeLaCombe** [comment]
The tabindex of -1 on the main container is to assist with the Skip to Content button provided with the navigation banner. If you are looking to remove that, I wonder if you can reprogram that button to instead attach an id to the main node on focus, and then set the href of the button to that id if it doesn't already exist.

Here is a page that details scroll behavior when negative indexes are used. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Comment by bjankord** [comment]
# Tech Design

I'm planning to test only setting `tabindex="-1"` on the main element when the user triggers the skip to content button and then removing it after we shift focus.
I'm also planning to implement similar functionality in the overlay as I we use tabindex="-1" in the overlay for similar purpose, to shift focus when an overlay is open.
